tests/cypress/tests/e2e/state-user-dashboard.spec.js

```diff
@@ -21,7 +21,9 @@ describe("state user creates a program", () => {
   });
 
   it("hydrates modal correctly", () => {
-    cy.findAllByRole("button", { name: "Edit Program" }).last().click();
+    cy.contains("tr", "program title")
+      .findByRole("button", { name: "Edit Program" })
+      .click();
```

Comment:
Question: What if there's more than one with that title? Should this do `findAllByRole` and then click the last one that matches?